src/locale.json

```diff
@@ -0,0 +1,67 @@
+{
+  "en-GB": {
+    "separator": "",
+    "suffixes": ["y", "z", "a", "f", "p", "n", "µ", "m", "", "k", "M", "B", "t", "q", "Q", "Z", "Y"],
+    "delimiters": {
+      "thousands": ",",
+      "decimal": "."
+    },
+    "currency": {
+      "symbol": "£"
+    }
+  },
+  "en-US": {
+    "separator": "",
+    "suffixes": ["y", "z", "a", "f", "p", "n", "µ", "m", "", "k", "M", "B", "t", "q", "Q", "Z", "Y"],
+    "delimiters": {
+      "thousands": ",",
+      "decimal": "."
+    },
+    "currency": {
+      "symbol": "$"
+    }
+  },
+  "es-ES": {
+    "separator": "",
+    "suffixes": ["y", "z", "a", "f", "p", "n", "µ", "m", "", "k", "mm", "b", "t", "q", "Q", "Z", "Y"],
+    "delimiters": {
+      "thousands": ".",
+      "decimal": ","
+    },
+    "currency": {
+      "symbol": "€"
+    }
+  },
+  "es-LA": {
```

Comment:
Is this one for representing **L**atin **A**merica? I only want to use language-country codes that are part of the standard ISO set. So if you want one for Chile, can we do `"es-CL"`? I found this list of codes online: https://www.fincher.org/Utilities/CountryLanguageList.shtml